datalake/backend/tests/test_dataset_versions_endpoint_logging.py

```diff
@@ -0,0 +1,93 @@
+import logging
+from json import dumps
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+
+from jsonschema import ValidationError  # type: ignore[import]
+from pynamodb.exceptions import DoesNotExist
+from pytest import mark
+
+from ..endpoints.dataset_versions.create import create_dataset_version
+from .utils import Dataset, any_dataset_id, any_s3_url, any_valid_dataset_type
+
+
+class LogTests(TestCase):
+    logger: logging.Logger
+
+    @classmethod
+    def setup_class(cls) -> None:
+        cls.logger = logging.getLogger("datalake.backend.endpoints.dataset_versions.create")
+
+    @mark.infrastructure
+    @patch("datalake.backend.endpoints.dataset_versions.create.sfn_client.start_execution")
+    def test_should_log_payload_and_sfn_response(self, start_execution_mock: MagicMock) -> None:
```

Comment:
"sfn" is not necessarily an abbreviation a newbie to the code base would know. I would either expand it or just remove it.